ext/bg/js/settings.js

```diff
@@ -433,6 +440,20 @@ function onMessage({action, params}, sender, callback) {
 }
 
 
+/*
+ * Text parsing
+ */
+
+function onParseMecabChanged(e) {
```

Comment:
I think this functionality should be in ```Backend.onOptionsUpdated```.

Reply:
Hmm, that's true. I guess settings profiles wouldn't work with this code.